xds/internal/balancer/clusterresolver/e2e_test/balancer_test.go

```diff
@@ -445,3 +446,64 @@ func (s) TestOutlierDetectionConfigPropagationToChildPolicy(t *testing.T) {
 		t.Fatalf("Timeout when waiting for child policy to receive its configuration")
 	}
 }
+
+// Test verifies that LB waits for child policy configuration update
+// inline on receipt of configuration update.
+func (s) TestChildPoliciesConfigUpdatedInline(t *testing.T) {
+	// Override the newConfigHook to ensure picker was updated.
```

Comment:
// Override the ClientConnUpdateHook to track when the picker is updated.

Reply:
Updated